.github/workflows/ci.yml

```diff
@@ -24,11 +24,12 @@ jobs:
   test-install-briefcase:
     name: Test Install Briefcase
     needs: pre-commit
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.runner-os }}
     timeout-minutes: 30
     strategy:
       fail-fast: false
       matrix:
+        runner-os: [ "ubuntu-latest", "windows-latest", "macos-latest" ]
```

Comment:
Thorough is good... but this repo runs over 80 configurations *before* this PR. Do we have to run every possible combination on every operating system?

Reply:
the limit is 256 i think :) but yeah....i'll take it back out. the tests did pass for all OSes though.